node.gypi

```diff
@@ -1,4 +1,26 @@
 {
+  # 'force_load' means to include the static libs into the shared lib or
+  # executable. Therefore, it is enabled when building:
+  # 1. The executable and it uses static lib (cctest and node)
+  # 2. The shared lib
```

Comment:
Not really a helpful comment, it merely states _what_ this block does, not _why_.